plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/elastic/VersionedElasticAgentExtension.java

```diff
@@ -26,6 +27,8 @@
 import java.util.Map;
 
 public interface VersionedElasticAgentExtension {
+    String getExtensionVersion();
```

Comment:
This was required to avoid calls to specific extensions for unsupported requests. This can be removed if we are delegating it always and versioned extension is taking care of not making a pluginRequest.